src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -325,7 +325,7 @@ export default class UniversalDiscoveryModule extends Component {
         const { activeTab } = this.state;
 
         if (activeTab === TAB_BOOKMARKS) {
-            this.onItemSelect(null);
+            this.closeCMP();
```

Comment:
It should be named like `this.closeContentMetaPreview`. Please, use meaningful method/function names.